lib/rex/mime/message.rb

```diff
@@ -15,8 +15,10 @@ class Message
 
   def initialize(data=nil)
     self.header = Rex::MIME::Header.new
-    self.parts  = []
-    self.bound  = "_Part_#{rand(1024)}_#{rand(0xffffffff)}_#{rand(0xffffffff)}"
+    self.parts = []
+    random_bound = "#{rand(0xffffffff)}#{rand(0xffffffff)}#{rand(0xffffffff)}#{rand(0xffffffff)}"
+    random_bound = random_bound[0..29]
```

Comment:
This would be a little easier to read and would avoid really short boundaries if it used `Rex::Text::rand_text_numeric`. Rex text is [already](https://github.com/rapid7/rex-mime/blob/master/rex-mime.gemspec#L26) a requirement so there shouldn't be any problems with using it.

```suggestion
    random_bound = Rex::Text::rand_text_numeric(20..30)
```

Reply:
Oh, great! I switched to this, although I used a straight `(30)` since firefox doesn't appear to randomize this at all.